tensorflow/core/kernels/conv_grad_ops_3d.cc

```diff
@@ -1355,15 +1355,16 @@ class Conv3DBackpropInputOp<GPUDevice, T> : public OpKernel {
     using se::dnn::AlgorithmDesc;
     using se::dnn::ProfileResult;
     AlgorithmConfig algorithm_config;
+    ProfileResult best_result;
```

Comment:
Would it make sense to keep the best_result and best_result_no_scratch definitions local inside the 'if' block? Duplicated across the #if/#else or shared, either one is fine.

Reply:
ok I've made them inside the blocks